package.json

```diff
@@ -9,10 +9,15 @@
     "autoprefixer": "^7.1.4",
     "babel-core": "^6.26.0",
     "babel-loader": "^7.1.2",
+    "browser-sync": "^2.18.13",
     "eslint": "^4.8.0",
-    "grunt": "^1.0.1",
+    "gulp": "^3.9.1",
     "node-sass": "^4.5.3",
+    "require-dir": "^0.3.2",
     "sass-loader": "^6.0.6",
     "webpack": "^3.6.0"
+  },
+  "dependencies": {
+    "gulp-cli": "^1.4.0"
```

Comment:
do we need the gulp-cli dependency inside the project? 
shouldn't that be a global install ? 
if not please use dev dependency instead